src/plugins/manager.js

```diff
@@ -158,4 +158,8 @@ export class Manager {
   async list (): Promise<PluginPath[]> {
     throw new Error('abstract method Manager.list')
   }
+
+  async handleNodeVersionChange () {
+    // user will override
```

Comment:
@dickeyxxx fixed in https://github.com/heroku/cli-engine/pull/188/commits/8b58b246a816e319643f80c7a9595fceb071b70d